cpp/ql/lib/semmle/code/cpp/models/interfaces/Throwing.qll

```diff
@@ -10,13 +10,47 @@ import semmle.code.cpp.Function
 import semmle.code.cpp.models.Models
 import semmle.code.cpp.models.interfaces.FunctionInputsAndOutputs
 
+newtype TException =
+  TSEHException() or
+  TCxxException()
+
+/**
+ * Functions with information about how an exception is throwwn or if one is thrown at all.
```

Comment:
```suggestion
 * Functions with information about how an exception is thrown or if one is thrown at all.
```

Reply:
Fixed

Reply:
I think I got everything. Moving to open the PR for official review.